packages/server/src/server.ts

```diff
@@ -39,13 +38,13 @@ const mainLogger = pino({
     }
   }
 });
+
 const logger = mainLogger.child({ name: 'rpc-server' });
 const register = new Registry();
-
 const relay: Relay = new RelayImpl(logger, register);
 const cors = require('koa-cors');
 const app = new Koa();
-const rpc = koaJsonRpc();
+const rpc = new KoaJsonRpc();
```

Comment:
Does using the custom implementation make any of the controller logic in this file moveable?
That is should anything here move to `koaJsonRPc/index.ts`? 
Don't have to do it in this PR but it's be good to know so we track that.

Reply:
I think that's doable. I've opened a separate issue to take care of this. Issue #530